src/main/java/in/projecteka/consentmanager/SecurityConfiguration.java

```diff
@@ -74,15 +74,17 @@ public SecurityWebFilterChain securityWebFilterChain(
             ServerSecurityContextRepository securityContextRepository) {
 
         final String[] whitelistedUrls = {"/**.json",
-                                          "/ValueSet/**.json",
-                                          "/users/verify",
-                                          "/users/permit",
-                                          "/sessions",
-                                          "/**.html",
-                                          "/**.js",
-                                          "/**.yaml",
-                                          "/**.css",
-                                          "/**.png"};
+                "/ValueSet/**.json",
+                "/patients/generateotp",
+                "/patients/verifyotp",
+                "/users/verify",
+                "/users/permit",
+                "/sessions",
+                "/**.html",
+                "/**.js",
+                "/**.yaml",
+                "/**.css",
+                "/**.png"};
```

Comment:
Please try to keep the same formatting as it was.

Reply:
done.